packages/medusa/src/repositories/customer-group.ts

```diff
@@ -1,5 +1,31 @@
-import { EntityRepository, Repository } from "typeorm"
+import { MedusaError } from "medusa-core-utils"
+import { EntityRepository, In, Repository } from "typeorm"
 import { CustomerGroup } from "../models/customer-group"
+import { CustomerBatchIds } from "../types/customer-groups"
 
 @EntityRepository(CustomerGroup)
-export class CustomerGroupRepository extends Repository<CustomerGroup> {}
+export class CustomerGroupRepository extends Repository<CustomerGroup> {
+  async deleteCustomerBatch(
+    groupId: string,
+    customerIds: CustomerBatchIds[]
```

Comment:
Would prefer if this is a string array instead of a higher level DTO

```suggestion
    customerIds: string[]
```